### PR TITLE
add employee-ubi

### PR DESCRIPTION
**Pull request by okwme**

### CODE REVIEW

File: ideas/employee-ubi.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Employee UBI
+## Every x blocks each AIB employee receives y tokens.
+
+# Description
+This chain could be used for a lot of different purposes. I think it should begin with just one feature: minting tokens for all employees at a standard rate. Once tokens are accumulating we can decide how and why we would use them. It might be good however to begin thinking about if the company is willing to exchange the tokens for goods or services (free lunch, swag, employee of the month, etc).
```
Comment (tac0turtle) [inline ideas/employee-ubi.md]: we should be opposed to using `I` in these descriptions.

Comment (okwme) [reply ideas/employee-ubi.md]: Wait, should or shouldn't? Should I remove the "I" or you mean you like it?

Comment (tac0turtle) [reply ideas/employee-ubi.md]: sorry remove I

Comment (okwme) [reply ideas/employee-ubi.md]: :+1:

**Review by tac0turtle — APPROVED** [review]

**Pull request merged**